### DISCUSSION TITLE
Creating a `pkg:content` URI for consistent blob hash representation

### DISCUSSION THREAD

**Discussion by lumjjb** [package-url/purl-spec#335] [Q&A]
Hashes are one of the common ways in which content blobs are identified. In most of the spec, there is a way of which this is encoded, either via a string like "sha256:..." or a json that looks like `{"alg":"sha256", "value":"..."}`. This requires understanding the spec for each data format that wants to express a content hash of a blob. 

Wondering if we can create a consistent way to define and consume this through a PURL. Maybe something like: 

```
pkg:content/<alg>/<value>
```

or some variation using parameters (not super fluent in what the best way to encode this is!)

**Comment by bureado** [comment]
Would the checksum qualifier satisfy this?

**Reply by pombredanne** [reply]
@lumjjb using qualifiers is the way to go here IMHO
